config/helm/ec2-metadata-test-proxy/templates/clusterrole.yaml

```diff
@@ -9,29 +9,4 @@ rules:
     - nodes
   verbs:
     - get
-    - patch
-    - update
-- apiGroups:
-    - ""
-  resources:
-    - pods
-  verbs:
-    - list
-- apiGroups:
-    - ""
-  resources:
-    - pods/eviction
-  verbs:
-    - create
-- apiGroups:
-    - extensions
-  resources:
-    - daemonsets
-  verbs:
-    - get
-- apiGroups:
-    - apps
-  resources:
-    - daemonsets
-  verbs:
-    - get
+    
```

Comment:
The above changes look unrelated?

Reply:
It is unrelated, but when I was copying pasting the ec2-metadata-test-proxy helm chart, I noticed that neither of the test fixture charts needed the permissions. Note that both of these are only test fixtures and the only reason they are in helm is to make the cleanup easier in the e2e test suite.

Reply:
yerah, makes sense :)